lumicks/pylake/force_calibration/calibration_models.py

```diff
@@ -22,6 +22,29 @@
 from typing import Callable
 
 
+def viscosity_of_water(temperature):
+    """Computes the viscosity of water at a particular temperature.
+
+    These equations come from section 3.7 from [7].
+
+    [7] Huber, M. L., Perkins, R. A., Laesecke, A., Friend, D. G., Sengers, J. V.,
+    Assael, M. J., & Miyagawa, K. (2009). New international formulation for the viscosity of H2O.
+    Journal of Physical and Chemical Reference Data, 38(2), 101-125.
+
+    Parameters
+    ----------
+    temperature : float
+        Temperature [Celsius]. Should be between -20°C and 110°C
+    """
```

Comment:
I recommend adding the return value unit somewhere (Pa*s)

Reply:
Added a return field.